tox.ini

```diff
@@ -7,7 +7,7 @@
 # Use tox_slow.ini for longer running tests.
 
 [tox]
-envlist = py35, py36, py37, py38, flake8
+envlist = py, flake8
```

Comment:
Please revert this change:
1. py is defined by default, no need to explicitly list it
2. it's actually the same for pyXX but it's a documentation of the "support matrix"